webhooks/framework.go

```diff
@@ -3,30 +3,31 @@ package webhooks
 import (
```

Comment:
Delete this entire webhooks package. Move all this logic directly into the corresponding functions in service/handlers.go